### PR TITLE
adding non pg94 worker to this repo. This is replacement for the work…

### PR DESCRIPTION
**Pull request by JoeNunnelley**
…er recipe that used to live in socrata-jenkins cookbook. @michaelb990  please take a gander.

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -2,9 +2,9 @@ source "https://supermarket.getchef.com"
 
 metadata
 
-cookbook 'ssh_known_hosts', git: 'git@github.com:socrata-cookbooks/ssh_known_hosts'
-cookbook 'sbt', git: 'git@github.com:socrata-cookbooks/sbt'
-cookbook 'java', git: 'git@github.com:socrata-cookbooks/java'
+cookbook 'ssh_known_hosts', git: 'git@github.com:socrata-cookbooks/ssh_known_hosts.git'
```
Comment (michaelb990) [inline Berksfile]: you don't need to do this

File: attributes/default.rb [hunk 2]
```diff
@@ -3,7 +3,14 @@
 default['jenkins']['master']['jvm_options'] = '-XX:MaxPermSize=256m -Dorg.apache.commons.jelly.tags.fmt.timeZone=America/Los_Angeles'
 
 default['rbenv']['group_users'] = ['ubuntu']
+
 default['java']['jdk_version'] = '8'
+default['java']['install_flavor'] = 'oracle'
```
Comment (michaelb990) [inline attributes/default.rb]: why? was this not working before?

File: recipes/_ruby.rb [hunk 3]
```diff
@@ -2,18 +2,37 @@
...
 rubies.each do |the_ruby|
   rbenv_gem 'bundler' do
     ruby_version the_ruby
   end
 end
+
+# Add packages recommended by ruby-build, for some reason these are not all installed by the cookbook.
+# Source: https://github.com/sstephenson/ruby-build/wiki#suggested-build-environment
+%w(
+  autoconf
```
Comment (michaelb990) [inline recipes/_ruby.rb]: these should be first, before the `rbenv_ruby` line. also, `rbenv::ruby_build` should add these already. let's first try removing them completely.

**Comment by michaelb990** [comment]
I think the tests failed because of the ruby dependency packages not being installed when we try and build the various rubies.

**Comment by michaelb990** [comment]
Tests are still failing.

**Comment by JoeNunnelley** [comment]
tests are passing now. merge?

**Comment by michaelb990** [comment]
This diff is wrong. There should be more, I think.

**Comment by JoeNunnelley** [comment]
due to a mistaken merge issue, I'm closing this one and opening a new fresh PR to get this change in.

**Pull request closed without merging**